input/templates/my-account-edit-address.json

```diff
@@ -0,0 +1,18 @@
+{
+	"header" : { "$ref" : "common/header.json" },
+	"footer" : { "$ref" : "common/footer.json" },
+	"content" : {
+		"contentTitle" : "Address Book",
+		"defaultAddressesTitle" : "Default Addresses",
+		"shippingAddress" : {
```

Comment:
I would imagine that you can edit any address, not only the shipping address. Therefore I would call this simply `address`.